package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "mockserver",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/mockserver.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@user-office-software/duo-logger": "^2.0.2",
+    "fs": "^0.0.1-security",
+    "fs.promises": "^0.1.2",
```

Comment:
Do we need this? I can't tell what it does from its [NPM page](https://www.npmjs.com/package/fs.promises), and I don't think we're using `Promise`s when reading files